package.json

```diff
@@ -1,31 +1,15 @@
 {
-  "name": "material-ui",
+  "name": "material-ui-build",
+  "private": true,
   "author": "Call-em-all Engineering Team",
   "version": "0.15.0-alpha.2",
   "description": "Material Design UI components built with React",
-  "main": "./lib",
-  "scripts": {
-    "build": "babel ./src --ignore *.spec.js --out-dir ./lib",
-    "build:icon-index": "cd src/svg-icons/ && babel-node index-generator.js",
-    "coverage": "npm run test",
-    "coverage:combine": "istanbul report --dir test/coverage/combined --include test/**/*coverage.json text-summary lcovonly json",
-    "lint": "eslint src docs/src test/browser test/unit && echo \"eslint: no lint errors\"",
-    "prebuild": "rimraf lib",
-    "prepublish": "in-publish && npm run build || not-in-publish",
-    "test": "npm run test:unit && npm run test:browser && npm run test:unit:coverage && npm run coverage:combine",
-    "test:browser": "npm run test:browser:base -- --single-run",
-    "test:browser:watch": "npm run test:browser:base -- --auto-watch",
-    "test:browser:base": "karma start test/karma.conf.js",
-    "test:unit": "babel-node test/unit.js",
-    "test:unit:watch": "babel-node test/unit.watch.js",
-    "test:unit:coverage": "babel-node node_modules/.bin/isparta cover --report lcovonly --include-all-sources --dir test/coverage/unit test/unit.js"
-  },
+  "main": "./src/index.js",
   "keywords": [
     "react",
     "react-component",
     "material design",
-    "material-ui",
-    "material ui"
```

Comment:
Woops, I thought this was a duplicate when I first read it, I didn't even see the `-` difference.